packages/plugin-typescript/sources/typescriptUtils.ts

```diff
@@ -0,0 +1,58 @@
+import {Request, Requester, Response} from '@algolia/requester-common';
```

Comment:
@haroenv @nunomaduro Maybe these should be re-exported in `algoliasearch`? 🤔

Reply:
We have no plans for that, sorry.